resources/lib/ui/embed_extractor.py

```diff
@@ -20,16 +26,29 @@ def load_video_from_url(in_url):
     except:
         raise
 
-    for extractor in _EMBED_EXTRACTORS.keys():
-        if in_url.startswith(extractor):
-            try:
-                return _EMBED_EXTRACTORS[extractor](url, page_content)
-            except Exception as e:
-                print "[*E*]: %s" % e
+    # Thanks to githubus11 for sorting function - unfortunately I can't get it working with OpenLoad involved
```

Comment:
Passing the setting variable for reversing the order for anime episodes was easy enough. However passing the setting variable sortres involves passing it through 5 or more functions before it gets to where it needs to go from default.py... Are you sure that's how you want it done?
Either way that's about all the time I can spend on this for now. Thanks for the comments :) I may get some time to work on this in future, but if you would like to implement it sooner feel free to make any changes you like.